content/concepts/content-routing/overview.md

```diff
@@ -4,4 +4,56 @@ description: "Learn about Noise in libp2p."
 weight: 218
 ---
 
-Coming soon!
+## Overview
+
+Content routing refers to the process of directing data to its intended recipient within a
+P2P network. In a traditional client-server network, routing is a straightforward process
+because a central authority manages the flow of data. However, in a P2P network, there is
+no central authority, so routing becomes more complex.
+
+One of the main challenges of content routing in P2P networks is scalability. As the number
+of peers in a network increases, the number of possible routes for data to travel also increases.
```

Comment:
data only travels point to point (except when relayed). The problem content routing solves is more about figuring out who has the data you are looking for. Not how the data goes from one node to another.